includes/routes/event/translations.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Wporg\TranslationEvents\Routes\Event;
+
+use Exception;
+use GP;
+use GP_Locales;
+use GP_Original;
+use Translation_Entry;
+use Wporg\TranslationEvents\Attendee_Repository;
+use Wporg\TranslationEvents\Event\Event_Repository_Interface;
+use Wporg\TranslationEvents\Routes\Route;
+use Wporg\TranslationEvents\Stats\Stats_Calculator;
+use Wporg\TranslationEvents\Translation_Events;
+
+/**
+ * Displays the event details page.
+ */
+class Translations_Route extends Route {
+	private $event_repository;
+
+	public function __construct() {
+		parent::__construct();
+		$this->event_repository = Translation_Events::get_event_repository();
+	}
+
+	public function handle( string $event_slug, string $locale, string $status = '%' ): void {
```

Comment:
It feels a bit weird to have the SQL wildcard here, initially it was confusing to me. Maybe this could be `all` instead?

Reply:
Understandable, I changed it to `any`.